java/core/page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/drivers/configure/WebDriverManagerConfigurator.java

```diff
@@ -0,0 +1,205 @@
+package ru.sbtqa.tag.pagefactory.drivers.configure;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.stream.JsonReader;
+import io.github.bonigarcia.wdm.Architecture;
+import io.github.bonigarcia.wdm.BrowserManager;
+import io.github.bonigarcia.wdm.OperativeSystem;
+import static io.github.bonigarcia.wdm.OperativeSystem.LINUX;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import static org.apache.commons.lang3.SystemUtils.IS_OS_LINUX;
+import static org.apache.commons.lang3.SystemUtils.IS_OS_MAC;
+import static org.apache.commons.lang3.SystemUtils.IS_OS_WINDOWS;
+import static org.openqa.selenium.remote.BrowserType.IE;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.pagefactory.support.properties.Configuration;
+import ru.sbtqa.tag.pagefactory.support.properties.Properties;
+
+public class WebDriverManagerConfigurator {
+
+    private static final Logger LOG = LoggerFactory.getLogger(WebDriverManagerConfigurator.class);
+
+    private static final Configuration PROPERTIES = Properties.getProperties();
+
+    private static final String MAPPING_FILES_PATH = "drivers/mapping/";
+    private static final String MAPPING_FILES_EXTENSION = ".json";
+
+    public static void configureDriver(BrowserManager webDriverManager, String browserType) {
+        if (!PROPERTIES.getDriversPath().isEmpty()) {
+            System.setProperty("webdriver." + browserType + ".driver", new File(PROPERTIES.getDriversPath()).getAbsolutePath());
```

Comment:
We will fail there again using org.openqa.selenium.remote.BrowserType.IE.
First arg of System.setProperty is going to be equals to "webdriver.internet explorer.driver" instead of "webdriver.ie.driver". Watch PR #108 in Page-Factory
I guess we need to create our own BrowserType class with static fields

Reply:
add ru.sbtqa.tag.pagefactory.support.BrowserType